python/lsst/ctrl/oods/gen3ButlerIngester.py

```diff
@@ -236,12 +234,8 @@ async def ingest(self, file_list):
         """
 
         # Ingest image.
-        try:
-            loop = asyncio.get_running_loop()
-            with concurrent.futures.ThreadPoolExecutor(max_workers=1) as pool:
-                await loop.run_in_executor(pool, self.task.run, file_list)
-        except Exception as e:
-            LOGGER.info("Ingestion failure: %s", e)
+        await asyncio.sleep(0)
```

Comment:
Not clear that this helps much, and we should prioritize ingestion ahead of other things.

Reply:
I'll take that out.